packages/db/src/lib/collection.ts

```diff
@@ -27,3 +29,56 @@ export const getCollectionsByGranuleIds = async (
     .groupBy(`${collectionsTable}.cumulus_id`);
   return await RetryOnDbConnectionTerminateError(query, {}, log);
 };
+
+// TODO - This function is going to be super-non-performant
+// We need to identify the specific need here and see if we can optimize
```

Comment:
Would it help to add some logging here so that if users encounter a bottleneck, we know more about the use case?

Reply:
I'm not entirely sure what that looks like, as the bottleneck is likely *in* the SQL server logs. 

I feel this probably falls *solidly* out of scope for this ticket, but open to talking about it.

Reply:
Nope, only asking in case we could quickly add logs here.